### PR TITLE
[MODORG-53] - BE - Implement CRUD for Organization Settings

### PR DESCRIPTION
**Pull request by imerabishvili**
## Purpose
https://issues.folio.org/browse/MODORG-53

## Approach
Create setting and collection of settings schemas.

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Examples exist for all schemas
  - [ ] Descriptions exist for all schema properties
  - [ ] All schemas pass raml-cop linting
- Does this introduce breaking changes?
  - [ ] Were there any schema changes?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: mod-orgs/examples/setting_post.sample [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "key": "BANKING_INFORMATION_ENABLED",
```
Comment (imerabishvili) [inline mod-orgs/examples/setting_post.sample:2]: Choosing between:
BANKING_INFORMATION_ENABLED: TRUE/FALSE
versus
BANKING_INFORMATION: ENABLED/DISABLED
WDYT?

Comment (SerhiiNosko) [reply mod-orgs/examples/setting_post.sample:2]: I think first option is more standardized and we can choose it, also lets value will be from lower case :
"BANKING_INFORMATION_ENABLED": true

Comment (SerhiiNosko) [reply mod-orgs/examples/setting_post.sample:2]: and collection.sample also should be corrected with lower case string in example

Comment (imerabishvili) [reply mod-orgs/examples/setting_post.sample:2]: done

File: mod-orgs/examples/setting_collection.sample [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "settings": [
+    {
+      "id": "f04c7277-0019-43cf-84b3-02d894a9d81a",
+      "name": "BANKING_INFORMATION_ENABLED",
+      "status": "true",
```
Comment (SerhiiNosko) [inline mod-orgs/examples/setting_collection.sample:6]: should be corrected, use key value instead of name and status

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**